swing-lib/src/main/java/org/triplea/swing/JTabbedPaneBuilder.java

```diff
@@ -10,7 +10,6 @@
 
 import org.apache.commons.math3.util.Pair;
```

Comment:
Wow. We include the whole commons-math dependency for this Pair utility when we have Tuple or could create a quick helper class?

Reply:
`Tuple` is no longer accessible from this code. Since this code is moved, if I modify it, then it would need to be completely re-scrutinized for everything and completely re-tested beyond just having been moved. The diff would also be impossible to tell what changed as git would show it as a completely new file.

A quick helper class I would agree would be a good refactoring to drop the dependency, but it's scope creep that would make the diff unreviewable.

Reply:
Would you mind creating a follow-up PR in this case addressing the minor remaining refactoring issues?
Also I think lombok is available in all subprojects, so using `@Data` will keep things rather clean.

Reply:
Side-note: data-beans, mutable state, and `@Data` generally should be avoided, ie:  `@Value` > `@Data`

> Would you mind creating a follow-up PR in this case addressing the minor remaining refactoring issues?

In some ways, yes. The updates in this PR should not be predicated on fixing existing problems that were intentionally left unchanged. Scope and intent of a PR is important.

Given we are already investing time on this area,  `Pair` and `Tuple` are both arguably anti-patterns and should be removed across the entire code-base, I looked at what it would take to remove the `Pair` usage.   I went ahead and removed `Pair` here: 3e869c16ae6e7264da74132cef144f37ed5de9a0 . After-the-fact, the benefit was not quite as much as I hoped. If that introduces new issues and more code review round-trips, I will come to regret the scope increase.